documentation/create_overview_doc/README.md

```diff
@@ -51,21 +51,27 @@ sh
 kt
 ```
 
+For flow map generation with Code2Flow, the tool currently supports the following languages, each with specific dependencies:
+
+- JavaScript: Requires Acorn for parsing. Ensure Acorn is installed and accessible in your environment.
+- Ruby: Uses Parser for parsing. Ensure the Parser gem is installed and accessible.
+- PHP: Employs PHP-Parser for parsing. PHP-Parser should be installed and accessible in your environment.
+- Python: No additional dependencies are required for Python projects.
+
+If your project uses a language not supported by Code2Flow for flow map generation, the documentation generation capabilities will still be available, but without the flow map visualization for those specific languages.
+
 ## Prerequisites
 
 Ensure the following tools are installed:
 
 - `bito` : https://github.com/gitbito/CLI
 
-- `mermaidcli` : https://github.com/mermaid-js/mermaid-cli
+- `Code2Flow:` : https://github.com/scottrogowski/code2flow
```

Comment:
**Suggestion**: Given the transition to Code2Flow, updating the 'Prerequisites' section of the README to include detailed steps for installing Code2Flow and its dependencies will help users set up their environment more efficiently. Additionally, consider linking to the official Code2Flow documentation for users seeking more comprehensive guidance. <br> **Code Suggestion**: 
 ```
 - 'Code2Flow': Ensure Code2Flow is installed by following the instructions at https://github.com/scottrogowski/code2flow. Code2Flow requires Python 3.6 or later.
 ```